.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# this image is built and updated weekly
+# https://github.com/retaildevcrews/codespaces-images
+FROM ghcr.io/retaildevcrews/k3d:latest
+
+# some images require specific values
+ARG USERNAME=vscode
+
+# [Optional] Uncomment this section to install additional OS packages.
+# alternatively, you can add commands to on-create.sh and/or post-create.sh
+# on-create runs before the Codespaces UI appears
+# post-create runs in the background after the UI appears
+# RUN apt-get -y install --no-install-recommends <your-package-list-here>
+
+# upgrade packages
+# you can improve startup time by runing in post-create.sh
+# RUN apt upgrade -
+
+# get install script and install node
+# [Choice] Node.js version: 16, 14, 12
+ARG VARIANT=14
+RUN curl -sL https://deb.nodesource.com/setup_${VARIANT}.x | sudo -E bash - && \
+    sudo apt-get install -y nodejs
```

Comment:
I think you  want an apt-get update before the install to make sure the packages are updated